openslides/users/urls.py

```diff
@@ -4,75 +4,10 @@
 
 urlpatterns = patterns(
     '',
-    # User
-    url(r'^$',
-        views.UserListView.as_view(),
-        name='user_list'),
-
-    url(r'^new/$',
-        views.UserCreateView.as_view(),
-        name='user_create'),
-
-    url(r'^new_multiple/$',
-        views.UserMultipleCreateView.as_view(),
-        name='user_create_multiple'),
-
-    url(r'^(?P<pk>\d+)/$',
-        views.UserDetailView.as_view(),
-        name='user_detail'),
-
-    url(r'^(?P<pk>\d+)/edit/$',
-        views.UserUpdateView.as_view(),
-        name='user_update'),
-
-    url(r'^(?P<pk>\d+)/del/$',
-        views.UserDeleteView.as_view(),
-        name='user_delete'),
-
-    url(r'^(?P<pk>\d+)/reset_password/$',
```

Comment:
The ResetPasswordView is still required because there is no possibility to reset the password via REST API.